Multisig/UI/Transaction/Execution/ReviewExecutionCellBuilder.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ReviewExecutionCellBuilder.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 11.01.22.
+//  Copyright © 2022 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+class ReviewExecutionCellBuilder: TransactionDetailCellBuilder {
```

Comment:
Do we need CellBuilder files, why don't we just create in the view controller directly

Reply:
Well, this extracts responsibility of building the cells into a separate class and leaves the view controller with gluing things together.

Also, this time this cell builder extends existing one, so it's easy to compose that one. If I would have the TxDetailCellBuilder code in the TxDetailsVC then I would need to inherit from the TxDetailsVC in my review execution controller, which I don't want because it has some other logic.